doc/jupyterlab/sharing.md

```diff
@@ -31,4 +31,4 @@ you with a unique URL that you can share.
 pip install jupysql
 ```
 
-> **For detailed instructions [click here.](https://docs.cloud.ploomber.io/en/latest/dashboards/jupyterlab-plugin.html)**
\ No newline at end of file
+> **For detailed instructions [click here.](https://docs.cloud.ploomber.io/en/latest/apps/jupyterlab-plugin.html)**
```

Comment:
this was already fixed in master, I think u need to rebase

Reply:
fixed